datajunction-clients/python/datajunction/client.py

```diff
@@ -239,18 +282,30 @@ def data(  # pylint: disable=too-many-arguments,too-many-locals
             poll_interval = 1  # Initial polling interval in seconds
             job_state = models.QueryState.UNKNOWN
             results = None
+            path = "/data/"
+            params = {
+                "dimensions": dimensions or [],
+                "filters": filters or [],
+                "engine_name": engine_name or self.engine_name,
+                "engine_version": engine_version or self.engine_version,
+                "async_": async_,
+            }
+
+            if (node_name and metrics) or not (node_name or metrics):
+                raise DJClientException(
```

Comment:
It looks like the client unit tests are failing due to lack of coverage of this line. Do you think you can add a small test for this?

Reply:
Yup, fixed.